### PR TITLE
setup instructions

### PR DESCRIPTION
**Pull request by chris-prener**
The contribution in this PR is based on discussion on Issue 135 in the `r-raster-vector-geospatial` repo - https://github.com/datacarpentry/r-raster-vector-geospatial/issues/135 . It also addresses issue #4 in this repo.

Set-up instructions include all required software, geospatial libraries, and R packages for both local installations and using Docker's `rocker/geospatial` image.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -1,6 +1,309 @@
...
+To install them from the `R` prompt, type:
+
+```r
+install.packages(c("devtools", "dplyr", "raster", "rgdal", "rasterVis", "sf"))
+```
+
+Additionally, you will need to install the *development* version of [`ggplot2`](https://github.com/tidyverse/ggplot2):
+
+```r
+devtools::install_github("tidyverse/ggplot2")
```
Comment (fmichonneau) [inline setup.md]: replace with `remotes::install_github(.....`

**Comment by ErinBecker** [comment]
Thanks for taking care of this @chris-prener. Please let me know if you would like to make those changes that @fmichonneau suggested or if you'd like me to go in and add those changes to your PR. Happy to help if I can.

File: setup.md [hunk 2]
```diff
@@ -1,6 +1,309 @@
...
+> Alternatively, participants who are comfortable with the command line can install the geospatial libraries individually using [homebrew](https://brew.sh):
+>
+>~~~
+>$ brew tap osgeo/osgeo4mac && brew tap --repair
+>$ brew install proj
+>$ brew install geos
+>$ brew install gdal2 --with-armadillo --with-complete --with-libkml --with-unsupported
+>$ brew link --force gdal2
+>~~~
+>{: .bash}
```
[the remark this answers is not in the document]
Comment (chris-prener) [reply setup.md]: completed in latest commit

File: setup.md [hunk 3]
```diff
@@ -1,6 +1,309 @@
...
+* [`dplyr`](https://cran.r-project.org/package=dplyr)
+* [`raster`](https://cran.r-project.org/package=raster)
+* [`rgdal`](https://cran.r-project.org/package=rgdal)
+* [`rasterVis`](https://cran.r-project.org/package=rasterVis)
+* [`sf`](https://cran.r-project.org/package=sf)
+
+To install them from the `R` prompt, type:
+
+```r
+install.packages(c("devtools", "dplyr", "raster", "rgdal", "rasterVis", "sf"))
```
[the remark this answers is not in the document]
Comment (chris-prener) [reply setup.md]: completed in latest commit

**Comment by chris-prener** [comment]
@fmichonneau and @ErinBecker - these were all made - https://github.com/chris-prener/geospatial-workshop/commit/62b0c04943af7097715581722ac69e7f1bb5937e

**Comment by ErinBecker** [comment]
Thanks @chris-prener!

**Pull request merged**